packages-exp/auth-exp/src/core/user/id_token_result.test.ts

```diff
@@ -0,0 +1,145 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { expect, use } from 'chai';
+import * as chaiAsPromised from 'chai-as-promised';
+import * as sinon from 'sinon';
+
+import { base64Encode, FirebaseError } from '@firebase/util';
+
+import { testUser } from '../../../test/mock_auth';
+import { User } from '../../model/user';
+import { getIdTokenResult } from './id_token_result';
+
+use(chaiAsPromised);
+
+const MAY_1 = new Date('May 1, 2020');
+const MAY_2 = new Date('May 2, 2020');
+const MAY_3 = new Date('May 3, 2020');
+
+function makeJWT(claims: object): string {
+  const b64 = base64Encode(JSON.stringify(claims));
+  return `algorithm.${b64}.signature`;
```

Comment:
Alex made the point later in the review that the middle part is often called payload. It's clear that this is base64 as well. Lets be consistent with whatever you choose!

Reply:
Done